src/Extracting/Strategies/GetFromInlineValidatorBase.php

```diff
@@ -51,7 +51,7 @@ public function lookForInlineValidationRules(ClassMethod $methodAst): array
                 break;
             } else if (
                 // Try $validator = Validator::make(...)
-                $rvalue instanceof Node\Expr\StaticCall && $rvalue->class->parts && end($rvalue->class->parts) == "Validator"
+                $rvalue instanceof Node\Expr\StaticCall && isset($rvalue->class->parts) && $rvalue->class->parts && end($rvalue->class->parts) == "Validator"
```

Comment:
empty = isset + empty

```suggestion
                $rvalue instanceof Node\Expr\StaticCall && !empty($rvalue->class->parts) && end($rvalue->class->parts) == "Validator"
```